### PR TITLE
adds GP2 resources modal

### PR DESCRIPTION
**Pull request by markcsims**

### CODE REVIEW

File: packages/gp2-components/src/organisms/ResourceModal.tsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+  Paragraph,
+  pixels,
+  utils,
+} from '@asap-hub/react-components';
+import { UrlExpression } from '@asap-hub/validation';
+import { css } from '@emotion/react';
+import { useState } from 'react';
+import { mobileQuery } from '../layout';
+
+const types = ['Link' as const, 'Note' as const];
```
Comment (jhonny-m) [inline packages/gp2-components/src/organisms/ResourceModal.tsx:18]: we should probably use the types from the model

File: apps/gp2-frontend/src/projects/ProjectDetail.tsx [hunk 2]
```diff
@@ -35,7 +36,13 @@ const ProjectDetail = () => {
           {isProjectMember && (
             <Route path={projects({}).project({ projectId }).resources({}).$}>
               <Frame title="Resources">
-                <ProjectResources {...project} />
+                <ProjectResources
+                  {...project}
+                  add={
+                    projects({}).project({ projectId }).resources({}).add({}).$
```
[the remark this answers is not in the document]
Comment (markcsims) [reply apps/gp2-frontend/src/projects/ProjectDetail.tsx:41]: Done

File: packages/gp2-components/src/organisms/ResourceModal.tsx (hunk 1 above)
Comment (markcsims) [reply packages/gp2-components/src/organisms/ResourceModal.tsx]: Done

File: apps/gp2-frontend/src/projects/ProjectDetail.tsx [hunk 3]
```diff
@@ -35,7 +35,12 @@ const ProjectDetail = () => {
...
```
Comment (jhonny-m) [inline apps/gp2-frontend/src/projects/ProjectDetail.tsx:41]: what about this add?

Comment (markcsims) [reply apps/gp2-frontend/src/projects/ProjectDetail.tsx:41]: In the next ticket

**Review by jhonny-m — APPROVED** [review]

**Pull request merged**